library/EasyPipeline/src/Easy.hs

```diff
@@ -36,59 +39,77 @@ import qualified Juvix.Frontend.Types.Base as Frontend
 import qualified Juvix.FrontendDesugar as FrontDesugar
 import Juvix.Library
 import qualified Juvix.Library.Feedback as Feedback
+import qualified Juvix.Library.HashMap as Map
 import qualified Juvix.Library.NameSymbol as NameSymb
 import qualified Juvix.Library.Sexp as Sexp
 import qualified Juvix.Pipeline as Pipeline
 import qualified Juvix.Pipeline.Compile as Compile
+import qualified Juvix.ToCore.Types as ToCore.Types
 import qualified Text.Pretty.Simple as Pretty
 import Prelude (error)
+import qualified Prelude (Show (..))
 
 --------------------------------------------------------------------------------
 -- OPTIONS
 --------------------------------------------------------------------------------
 
-data Options = Opt
+instance Prelude.Show (Core.Parameterisation primTy primVal) where
+  show _ = "param"
```

Comment:
Should we define a proper instance in Core? Apparently we never had a use for it, so maybe not.